src/main/java/com/kmush12/CurrencyConverter/exchange/ExchangeRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.kmush12.CurrencyConverter.exchange;
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ExchangeRepository extends MongoRepository<ExchangeRate, String> {
+    ExchangeRate findByOriginCurrency(String originCurrency);
```

Comment:
Tu raczej zwróć `Optional<ExchangeRate>` bo jak nie znajdzie Ci nic po tym co wpiszesz to zwróci nulla.